addons/spreadsheet/static/src/pivot/pivot_model.js

```diff
@@ -550,6 +610,11 @@ export class SpreadsheetPivotModel extends PivotModel {
             } else {
                 value = toNormalizedPivotValue(field, groupValue, aggregateOperator);
             }
+
+            if (this._isDateField(field) && !isPositional) {
```

Comment:
can be set as an else if in the previous statement? Don't know if it's clearer though